### ISSUE TITLE
Question/Feature: Does this work with latest version macOS 13.1 to 13.5

### ISSUE THREAD

**Issue by RLNY88** [dan-snelson/Nudge-Post-install#5]
Hi All,

Hope all is well. Thanks in advance for looking into this. Much appreciated.

**Describe the solution you'd like**
Nudge Minimum OSVersion 13.4.1 and below to upgrade to the latest macOS version 13.5.

Best, RL

**Comment by dan-snelson** [comment]
@RLNY88:

Will you please restate your request?

(We’re prompting users to upgrade to macOS `13.5`.)

**Comment by RLNY88** [comment]
@dan-snelson 
Thanks for the quick reply. Sorry, let me rephrase...

I changed "requiredMinimumOSVersion" key and set it to 13.99 (see below). And it doesn't seem to detect Required OS Version (see screenshot below). I hope you can point me to the right direction.

    <key>requiredInstallationDate</key>
            <date>2023-01-28T00:00:00Z</date>
            <key>requiredMinimumOSVersion</key>
            <string>13.99</string>

![Required OS Version](https://github.com/dan-snelson/Nudge-Post-install/assets/118987813/1cb2850f-8268-473d-b842-017ff5219bba)

**Comment by dan-snelson** [comment]
@RLNY88:

From your screenshot, you're executing Nudge in `Demo Mode`.

If you haven't already, please join the 1,600+ Mac Admins on the [#nudge-jamf](https://slack.com/app_redirect?channel=C02KPHZ7MDX) Channel.

Closing.
